notebooks/cvr_load.jl

```diff
@@ -1,14 +1,15 @@
 ### A Pluto.jl notebook ###
-# v0.16.1
+# v0.19.40
 
 using Markdown
 using InteractiveUtils
 
 # This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
 macro bind(def, element)
     quote
+        local iv = try Base.loaded_modules[Base.PkgId(Base.UUID("6e696c72-6542-2067-7265-42206c756150"), "AbstractPlutoDingetjes")].Bonds.initial_value catch; b -> missing; end
```

Comment:
Did you add this, or did pluto do this automatically?
I'm just wondering why this extra check is necessary now but wasn't before?